manifests/controllerconfig.crd.yaml

```diff
@@ -837,6 +837,47 @@ spec:
                   proper way. Nobody is also changing this once the cluster is up
                   and running the first time, so, disallow regeneration if this changes.'
                 type: string
+              mtuMigration:
```

Comment:
Instead of this being a new field in the controllerconfig, is it possible to handle this under some networking status? e.g. using ccSpec.NetworkType somehow?

Reply:
I guess we could deprecate `ccSpec.NetworkType` and have instead `ccSpec.Network.Type` and `ccSpec.Network.MTUMigration`.

Reply:
+1 on @yuqi-zhang suggestion.

Reply:
`ccSpec.NetworkType` is a string. Unless we do some json encoding that would also complicate the templates I don't see how we can do that. Is that what you mean?